doc/WebSite/Unit-Of-Work.md

```diff
@@ -1,26 +1,26 @@
 ### Introduction
 
 Connection and transaction management is one of the most important
-concepts in an application that uses a database. When to open a
-connection, when to start a transaction, how to dispose the connection
-and so on.. ASP.NET Boilerplate manages database connection and
-transactions by it's **unit of work** system.
+concepts in an application that uses a database. You need to know when to open a
+connection, when to start a transaction, and how to dispose the connection,
+and so on... ASP.NET Boilerplate manages database connections and
+transactions by using its **unit of work** system.
 
 ### Connection & Transaction Management in ASP.NET Boilerplate
 
 ASP.NET Boilerplate **opens** a database connection (it may not be
-opened immediately, but opened in first database usage, based on ORM
+opened immediately, but opened during the first database usage, based on the ORM
 provider implementation) and begins a **transaction** when **entering**
-a **unit of work method**. So, you can use connection safely in this
-method. At the end of the method, the transaction is **commited** and
-the connection is **disposed**. If the method throws any **exception**,
-transaction is **rolled back** and the connection is disposed. In this
+a **unit of work method**. You can use the connection safely in this
+method. At the end of the method, the transaction is **committed** and
+the connection is **disposed**. If the method throws an **exception**, the
+transaction is **rolled back**, and the connection is disposed. In this
 way, a unit of work method is **atomic** (a **unit of work**). ASP.NET
-Boilerplate does all of these automatically.
+Boilerplate does all of this automatically.
 
-If a unit of work method calls another unit of work method, both uses
-same connection & transaction. The first entering method manages
-connection & transaction, others use it.
+If a unit of work method calls another unit of work method, both use the
+same connection & transaction. The first entered method manages the
+connection & transaction and then the others use reuse it.
```

Comment:
@beriniwlew what do you think about this part "others use reuse it" ? Is this a correct usage ?

Reply:
Woops